packages/mobile/src/utils/contacts.ts

```diff
@@ -1,18 +1,53 @@
-import { getMinimal, MinimalContact } from 'react-native-contacts'
-import { checkContactsPermission } from 'src/utils/androidPermissions'
+import { Platform } from 'react-native'
+import { getAll, getMinimal, MinimalContact } from 'react-native-contacts'
 import Logger from 'src/utils/Logger'
+import { checkContactsPermission } from 'src/utils/permissions'
 
 const TAG = 'utils/contacts'
 
+// Stop gap solution since getMinimal is not yet implement on iOS
```

Comment:
Please open an issue to track this. We found that on android it gave a 3-5x import speedup :)

Reply:
Yep, opened #1246